### DISCUSSION TITLE
Directory of community plugins

### DISCUSSION THREAD

**Discussion by Willyfrog** [mattermost/community#151] [General Ideas]
From this thinkaton there are a few ideas that get the response "there is a plugin that does that (or partially does that)" which seems to be showing that most people don't know about those/are hidden in someone's github gathering dust.

While having a Mattermost approval should require some minimal quality, maybe we could have a directory where people could easily send their plugins and could be browsed/searched/classified. There should be an obvious disclaimer if mattermost hasn't approved the plugin as "safe for use".

one good place to add a link to such directory is in the marketplace if the user gets to the end of the list of plugins and if the user performs a search and there are no results: "Hey, we couldn't find anything, but there is a giant list of community contributed plugins where you might find what you need"

**Comment by chetanyakan** [comment]
This is a really good idea.
This could be under a setting in the Marketplace.
Like `Enable community plugins awaiting approval from Mattermost`

**Comment by BenLloydPearson** [comment]
This is something I would also like for potential content sources. I.e. we can use stuff like this to create blog posts and code examples that show how to get the most out of Mattermost.

A [cursory search](https://github.com/search?q=org%3Amattermost+mattermost) on GitHub returns more than 2,400 results, and we've highlighted community plugins in many of our release announcement blog post. I'm planning to begin compiling a list of good examples that we can use for something like this, and I'll share it here when I have something to show.
